### ISSUE TITLE
Constant "Verify that the daemon is running" error

### ISSUE THREAD

**Issue by KuuminKochi** [OpenTabletDriver/OpenTabletDriver#2633]
### Acknowledgements

- [X] I have installed OpenTabletDriver by following its [official installation instructions](https://opentabletdriver.net/).
- [X] I have checked the [OpenTabletDriver Wiki](https://opentabletdriver.net/Wiki) and respective FAQ pages, and my issue was not covered or actually fixed.
- [X] I have searched the existing issues and this new issue is not a duplicate of any.
- [X] I have written a concise and meaningful title.

### Steps to Reproduce

On Gentoo:
1. Install OpenTabletDriver
2. Run OpenTabletDriver
3. Wait on the "Connecting to OpenTabletDriver Daemon..." screen
4. After a while, you'll get the error
![image](https://user-images.githubusercontent.com/97841709/217280366-f7111a33-9b4b-4932-9116-8343a1d7e09f.png)

### Expected Behavior

OpenTabletDriver starts up normally without any hassle

### Observed Behavior

"Verify that the daemon is running" error 

### Tablet Device

Huion H950P

### Diagnostic Information

![image](https://user-images.githubusercontent.com/97841709/217281032-fcb1e586-af3e-4839-b259-069e24e764a9.png)

**Comment by X9VoiD** [comment]
I'm not 100% sure how OTD is packaged for Gentoo, but if you're using systemd, enable `opentabletdriver.service`. If it's not there, you may adhoc your own solution by running `otd` on boot.
